### PR TITLE
Cleaning up PublicMaps XML

### PR DESCRIPTION
**Pull request by TheRealPear**
As part of a recent update to PGM that added the ability to flag any unused XML nodes and potential typos, I decided to start with PublicMaps as it receives far less updates compared to other repos.

~~Based on what I've seen so far, it seems like PGM flags `<edition>` as unused, yet several maps still has it. I'm not sure what the best solution would be here, probably to leave it alone or comment it out so it won't show up in console.~~ Resolved in PGMDev/PGM#1243

I've also borrowed some Warzone changes to make some of these maps use proto 1.4.2 for easier maintenance.

### CODE REVIEW

**Comment by arcadeboss** [comment]
Big Clean, indeed.

File: ctw/standard/hadron/map.xml [hunk 1]
```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0"?>
 <map proto="1.3.0">
 <name>Hadron</name>
-<version>1.0.8</version>
+<version>1.0.10</version>
```
Comment (CrazyisCreeps) [inline ctw/standard/hadron/map.xml:4]: Shouldn't this be 1.0.9?

File: dtcm/standard/fortress_of_an_easterly_battle/map.xml [hunk 2]
```diff
@@ -1,7 +1,6 @@
-<?xml version="1.0"?>
-<map proto="1.4.0">
+<map proto="1.4.2">
 <name>Fortress of an Easterly Battle</name>
-<version>1.0.2</version>
+<version>1.0.4</version>
```
Comment (CrazyisCreeps) [inline dtcm/standard/fortress_of_an_easterly_battle/map.xml:3]: 1.0.3?

File: flag/standard/hekla/map.xml [hunk 3]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0"?>
-<map proto="1.4.0">
+<map proto="1.4.2">
 <name>Hekla</name>
 <version>1.0.3</version>
```
Comment (calcastor) [inline flag/standard/hekla/map.xml:3]: This should be bumped, probably just missed in merge from main after Pugzy updated it

File: flag/standard/harb_kotf/map.xml [hunk 4]
```diff
@@ -121,15 +121,7 @@
...
-    <flag id="flag"
-          name="Flag"
-          color="white"
-          shared="true"
-          points-rate="1"
-          pickup-kit="flag-kit"
-          drop-kit="reset-health"
-          flag-proximity-metric="none"
-          show-respawn-on-pickup="true">
+    <flag id="flag" name="Flag" color="white" shared="true" points-rate="1" pickup-kit="flag-kit" drop-kit="reset-health" flag-proximity-metric="none" show-respawn-on-pickup="true">
```
Comment (calcastor) [inline flag/standard/harb_kotf/map.xml:124]: I've usually left each attribute line by line in KotF just for legibility reasons, but it's not a big deal

File: ctw/halloween/jungle_spook/map.xml [hunk 5]
```diff
@@ -47,7 +47,7 @@
 </kits>
 <spawns>
     <default>
-        <region yaw="0">
+        <region>
```
[the remark this answers is not in the document]
Comment (TheRealPear) [reply ctw/halloween/jungle_spook/map.xml:50]: [Yes, that's the default value.](https://pgm.dev/docs/modules/mechanics/spawns#regions-element-attributes)

File: flag/standard/harb_kotf/map.xml (hunk 4 above)
Comment (CrazyisCreeps) [reply flag/standard/harb_kotf/map.xml:124]: I mean you could argue that most of these changes are just between different ways of doing the same thing.

**Review by calcastor — APPROVED** [review]

**Pull request merged**